Harmonise icons

respencer opened the pull request:
# Description & Issue number it closes 

Harmonise icons to make more sense, and be more consistent.

Closes #6884

## Screenshots (if appropriate)

![Screenshot 2024-04-19 at 21-11-10 ChurchCRM Welcome to](https://github.com/ChurchCRM/CRM/assets/201834/32701125-b708-47ed-a746-972c296e9819)

![Screenshot 2024-04-19 at 21-06-15](https://github.com/ChurchCRM/CRM/assets/201834/da431648-96b2-4b03-a4af-bf33d0fb61e4)

![Screenshot 2024-04-19 at 21-12-26 ChurchCRM Sunday School Dashboard](https://github.com/ChurchCRM/CRM/assets/201834/6e871037-950e-4d0c-972a-c23643f4dd18)

![Screenshot 2024-04-19 at 21-07-07 ChurchCRM Sunday School Dashboard](https://github.com/ChurchCRM/CRM/assets/201834/5006cd8d-ebfa-4f80-a914-9ecc44c6499c)

## How to test the changes?

Update and go to:

/v2/dashboard

and

/sundayschool/SundaySchoolDashboard.php

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Visual inspection.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

respencer commented:
Found another dashboard that needs updating. Can I add that change to this commit too?

DAcodedBEAT commented:
> Found another dashboard that needs updating. Can I add that change to this commit too?

@respencer please do and take screenshots :)

DawoudIO commented:
Looks good, when you are ready let me know and I'll give it a quick review

respencer commented:
@DawoudIO - done.

Fixed People Dashboard icons:

![Screenshot 2024-04-22 at 21-55-18 ChurchCRM People Dashboard](https://github.com/ChurchCRM/CRM/assets/201834/2e56998e-460c-4a12-9106-11fd4fab25f6)

![Screenshot 2024-04-22 at 21-55-01 ChurchCRM People Dashboard](https://github.com/ChurchCRM/CRM/assets/201834/7f367d68-9fd1-4356-89a6-54ac75bd90d1)

Fixed newsletter icon:

![Peek 2024-04-22 22-00](https://github.com/ChurchCRM/CRM/assets/201834/340dceb2-d745-426f-b6cb-f750553a6520)

![Screen Shot 2024-04-22 at 22 02 41](https://github.com/ChurchCRM/CRM/assets/201834/f23ee7f9-d063-4371-9272-e38421c50036)

grayeul commented:
LGTM
